assets/dev/js/editor/utils/control-conditions.js

```diff
@@ -0,0 +1,166 @@
+import Conditions from './conditions';
+
+/**
+ * Control Conditions Class
+ *
+ * This class Handles conditions checks specifically related to element controls.
+ *
+ * @since 3.7.0
+ */
+export default class ControlConditions extends Conditions {
+	/**
+	 * Get Operator
+	 *
+	 * Returns the condition's comparison operator according to the structure of the condition and control values.
+	 *
+	 * @since 3.7.0
+	 *
+	 * @param conditionValue
+	 * @param isNegativeCondition
+	 * @param controlValue
+	 * @returns {string}
+	 */
+	getOperator( conditionValue, isNegativeCondition, controlValue ) {
```

Comment:
I don't see a point to moving it since it is only used for control conditions. It is only used in the `convertConditionToConditions()` method in this class.

Reply:
It can be useful in the admin side too

Reply:
Moved to `conditions.js`